pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.580.1</version>
```

Comment:
Which is the minimal required version? Currently the plugin works well on 1.424.x..1.532.x and 1.609.x LTS lines only, so bumping the version may impact the existing installations. In particular I'd prefer to support 1.509.3 at least